src/components/lablup-loading-spinner.ts

```diff
@@ -33,22 +33,54 @@ export default class LablupLoadingSpinner extends LitElement {
     return [
       // language=CSS
       css`
-        wl-progress-spinner {
```

Comment:
If you got rid of `wl-progress-spinner`, please remove[ importing statement](https://github.com/lablup/backend.ai-webui/pull/1088/files#diff-1dcfe4880af082ebb2cb8fc2c709fb67c57f3f85427fff91c827b76f00e58ca7L7) above.

Reply:
I remain some spinner code where new loading dots for grid can't replace it.